# Hardcoded path need single quoting. Otherwise if the conda env directory
# contains special chars like '@' (as in Galaxy) this leads to problems


--- a/util/gff3_introns.pl.in	2020-03-12 20:33:54.000000000 +0100
+++ b/util/gff3_introns.pl.in	2020-06-18 12:53:37.792710701 +0200
@@ -21,9 +21,9 @@
 	if (0) {
 	    $FP = new IO::File(">&STDOUT");
 	} elsif (defined($opt_D)) {
-	    $FP = new IO::File("| @BINDIR@/get-genome -D $opt_D -d $opt_d > get-genome.out");
+	    $FP = new IO::File('| @BINDIR@/get-genome -D $opt_D -d $opt_d > get-genome.out');
 	} else {
-	    $FP = new IO::File("| @BINDIR@/get-genome -d $opt_d > get-genome.out");
+	    $FP = new IO::File('| @BINDIR@/get-genome -d $opt_d > get-genome.out');
 	}
 
 	$gene_name = "";

--- a/util/gff3_splicesites.pl.in	2020-03-12 20:33:54.000000000 +0100
+++ b/util/gff3_splicesites.pl.in	2020-06-18 12:53:19.168963936 +0200
@@ -21,9 +21,9 @@
 	if (0) {
 	    $FP = new IO::File(">&STDOUT");
 	} elsif (defined($opt_D)) {
-	    $FP = new IO::File("| @BINDIR@/get-genome -D $opt_D -d $opt_d > get-genome.out");
+	    $FP = new IO::File('| @BINDIR@/get-genome -D $opt_D -d $opt_d > get-genome.out');
 	} else {
-	    $FP = new IO::File("| @BINDIR@/get-genome -d $opt_d > get-genome.out");
+	    $FP = new IO::File('| @BINDIR@/get-genome -d $opt_d > get-genome.out');
 	}
 
 	$gene_name = "";


--- a/util/gmap_build.pl.in	2020-06-01 02:24:13.000000000 +0200
+++ b/util/gmap_build.pl.in	2020-06-18 12:50:12.031880110 +0200
@@ -14,7 +14,7 @@
 Getopt::Long::Configure(qw(no_auto_abbrev no_ignore_case_always));
 
 # Default values
-my $bindir = "@BINDIR@";   # dirname(__FILE__)
+my $bindir = '@BINDIR@';   # dirname(__FILE__)
 my $sampling = 3;
 my $sleeptime = 2;
 
@@ -92,7 +92,7 @@
     print STDERR "Writing files under $user_gmapdb\n";
     $gmapdb = $user_gmapdb;
 } else {
-    $gmapdb = "@GMAPDB@";
+    $gmapdb = '@GMAPDB@';
     print STDERR "Destination directory not defined with -D flag, so writing files under $gmapdb\n";
 }
 


--- a/util/gmap_cat.pl.in	2020-04-22 20:17:57.000000000 +0200
+++ b/util/gmap_cat.pl.in	2020-06-18 12:51:58.162137863 +0200
@@ -3,9 +3,9 @@
 
 use warnings;	
 
-my $gmapdb = "@GMAPDB@";
+my $gmapdb = '@GMAPDB@';
 my $package_version = "@PACKAGE_VERSION@";
-my $bindir = "@BINDIR@";   # dirname(__FILE__)
+my $bindir = '@BINDIR@';   # dirname(__FILE__)
 
 
 use IO::File;


--- a/util/gtf_introns.pl.in	2020-03-12 20:33:54.000000000 +0100
+++ b/util/gtf_introns.pl.in	2020-06-18 12:52:58.717248919 +0200
@@ -21,9 +21,9 @@
 	if (0) {
 	    $FP = new IO::File(">&STDOUT");
 	} elsif (defined($opt_D)) {
-	    $FP = new IO::File("| @BINDIR@/get-genome -D $opt_D -d $opt_d > get-genome.out");
+	    $FP = new IO::File('| @BINDIR@/get-genome -D $opt_D -d $opt_d > get-genome.out');
 	} else {
-	    $FP = new IO::File("| @BINDIR@/get-genome -d $opt_d > get-genome.out");
+	    $FP = new IO::File('| @BINDIR@/get-genome -d $opt_d > get-genome.out');
 	}
 
 	@exons = ();


--- a/util/gtf_splicesites.pl.in	2020-03-12 20:33:54.000000000 +0100
+++ b/util/gtf_splicesites.pl.in	2020-06-18 12:52:34.533595600 +0200
@@ -21,9 +21,9 @@
 	if (0) {
 	    $FP = new IO::File(">&STDOUT");
 	} elsif (defined($opt_D)) {
-	    $FP = new IO::File("| @BINDIR@/get-genome -D $opt_D -d $opt_d > get-genome.out");
+	    $FP = new IO::File('| @BINDIR@/get-genome -D $opt_D -d $opt_d > get-genome.out');
 	} else {
-	    $FP = new IO::File("| @BINDIR@/get-genome -d $opt_d > get-genome.out");
+	    $FP = new IO::File('| @BINDIR@/get-genome -d $opt_d > get-genome.out');
 	}
 
 	@exons = ();


--- a/util/gtf_transcript_splicesites.pl.in	2020-03-12 20:33:54.000000000 +0100
+++ b/util/gtf_transcript_splicesites.pl.in	2020-06-18 12:53:52.012521236 +0200
@@ -21,9 +21,9 @@
 	if (0) {
 	    $FP = new IO::File(">&STDOUT");
 	} elsif (defined($opt_D)) {
-	    $FP = new IO::File("| @BINDIR@/get-genome -D $opt_D -d $opt_d > get-genome.out");
+	    $FP = new IO::File('| @BINDIR@/get-genome -D $opt_D -d $opt_d > get-genome.out');
 	} else {
-	    $FP = new IO::File("| @BINDIR@/get-genome -d $opt_d > get-genome.out");
+	    $FP = new IO::File('| @BINDIR@/get-genome -d $opt_d > get-genome.out');
 	}
 
 	@exons = ();

